packages/presentation/lib/utils/colors.dart

```diff
@@ -0,0 +1,4 @@
+class CustomColors {
+  static const gradientTopColor = 0xFFE51937;
+  static const gradientBottomColor = 0xFFDB5252;
```

Comment:
naming - what if we will have gradient with another colors at another screen. need to specify name